WebContent/WEB-INF/jsp/usersProfiles.jsp

```diff
@@ -169,6 +169,8 @@
 
         if(watchlists != null) {
             for (i=0; i<watchlists.length; i++) {
+                if(watchlists[i].name == '<fmt:message key="common.newName"/>') // skip unnamed lists
```

Comment:
This condition is in at least 3 places in this file, you can move it to a separate function:
"isUnnamed"